packages/api/src/middleware/adminMiddleware.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response } from 'express';
+import { Admin } from '@hangar/database';
+
+export const adminMiddleware = async (req: Request, res: Response, next: NextFunction) => {
+  const adminQuery = { user: req.user?.id };
```

Comment:
This is a relatively simply query object; can we just pass this in on line 9 and save the extra line?